cmd/root.go

```diff
@@ -39,26 +37,8 @@ var (
 
 		SilenceUsage: true,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			logLevel := log.InfoLevel
-			if verbose, _ := cmd.Flags().GetBool("verbose"); verbose {
-				logLevel = log.DebugLevel
-			}
-			logHandler, err := log.NewZap("console")
-			if err != nil {
-				return err
-			}
-			logger = log.NewLogger(
-				logHandler,
-				log.WithLevel(logLevel),
-			)
-			rpclog.ReplaceGrpcLogger(logger)
-			ctx = log.NewContext(ctx, logger)
-
 			exportCfg.DevIDPrefix, _ = cmd.Flags().GetString("dev-id-prefix")
-			exportCfg.EUIForID, _ = cmd.Flags().GetBool("set-eui-as-id")
-			ctx = export.NewContext(ctx, exportCfg)
-
-			cmd.SetContext(ctx)
```

Comment:
The context was previously never propagated to the subcommands and hence this logger and the exportconfig were never used.

I've fixed the export config but we don't need this logger since we use Zap in the source.